app/brave_main_delegate.cc

```diff
@@ -210,76 +188,13 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   command_line.AppendSwitchASCII(variations::switches::kVariationsServerURL,
                                  kVariationsServerURL.c_str());
 
-  // Enabled features.
```

Comment:
JFTR: I've checked all enabled features one by one to confirm all were properly migrated and that seems to be the case :+1: